test/suite/hook_tiered.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# [TEST_TAGS]
+# ignored_file
+# [END_TAGS]
+
+# hook_tiered.py
+#
+# Substitute tiered tables for regular (row-store) tables in Python tests.
+#
+# These hooks can be used to run the existing cursor tests on tiered tables.
+# They identify tests that create row-store tables and create tiered tables
+# instead.  The hook takes an optional argument to specify how many tiers
+# to create. The default is 2.
+#
+# To run with 3 tiers per table:
+#     ../test/suite/run.py --hooks tiered=3 cursor
+#
+# The hooks work with may other tests in the python suite but also encounter
+# a variety of failures that I haven't tried to sort out.
+from __future__ import print_function
+
+import os, sys, wthooks
+import unittest
+from wttest import WiredTigerTestCase
+
+# Print to /dev/tty for debugging, since anything extraneous to stdout/stderr will
+# cause a test error.
+def tty(s):
+    WiredTigerTestCase.tty(s)
+
```

Comment:
Should be removed, since it's no longer being used.

Reply:
Yup.  I took out my debugging messages, but forgot the method that prints them  :-)